.github/workflows/.deploy.yaml

```diff
@@ -1,4 +1,5 @@
-name: Deploy
+name: Deploy PR
+run-name: Deploy PR-${{ github.event.inputs.pr-number }}
```

Comment:
So instead of the current names:
![image](https://github.com/bcgov/common-hosted-form-service/assets/35532993/327a12a7-65c0-4d68-99b4-9d43876992c6)

We will have:
![image](https://github.com/bcgov/common-hosted-form-service/assets/35532993/164091df-a6ed-430d-9d4b-5dd1800877aa)